sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -76,11 +84,19 @@
             <footer-buttons
               :processing="processing"
               primaryLabel="Resolve restriction"
-              :secondaryLabel="restrictionData.isActive ? 'Cancel' : 'Close'"
+              :secondaryLabel="
+                !restrictionData.isActive ||
```

Comment:
looks like this logic is repeated multiple times, can we add a computed property and use

Reply:
Same as https://github.com/bcgov/SIMS/pull/1598#discussion_r1038512166 thanks!